src/org/usfirst/frc4904/robot/subsystems/CubeIO.java

```diff
@@ -0,0 +1,26 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class CubeIO extends Subsystem {
+
+	public static Motor roller;
+	public static final double INTAKE_SPEED = 1.0;
+	public static final double OUTTAKE_SPEED = -1.0;
+	public static final double AT_REST = 0.0;
+	
+	public CubeIO(Motor roller) {
+		this.roller = roller;
+	}
+	
+	@Override
+	protected void initDefaultCommand() {
```

Comment:
The default command for a subsystem should be something that either doesn't move or reverts the subsystem to to a safe neutral state.

Reply:
Also what @HowardStark said previously still stands: https://github.com/RoboticsTeam4904/2018-Code/pull/6#discussion_r160301428